cookiecutter.json

```diff
@@ -17,22 +17,10 @@
   "windows": "n",
   "use_pycharm": "n",
   "use_docker": "n",
-  "postgresql_version": [
-    "11.3",
-    "10.8",
-    "9.6",
-    "9.5",
-    "9.4"
-  ],
-  "js_task_runner": [
-    "None",
-    "Gulp"
-  ],
-  "cloud_provider": [
-    "AWS",
-    "GCP",
-    "None"
-  ],
+  "postgresql_version": ["11.3", "10.8", "9.6", "9.5", "9.4"],
+  "js_task_runner": ["None", "Gulp"],
+  "cloud_provider": ["AWS", "GCP", "None"],
```

Comment:
Could you please not change the lines wrapping and stick to the code style of the file? We tend to have 1 option per line as it's making diff easier to process.